### ISSUE TITLE
Some basic failed SQL cases

### ISSUE THREAD

**Issue by zhixingheyi-tian** [oap-project/gluten#523]
**Describe the bug**
Find some basic failed SQL cases when validating Gluten.

**To Reproduce**
I have created two ignore UTs in https://github.com/oap-project/gluten/blob/main/backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```
ignore("Test isnull function") {
    val df = runQueryAndCompare("SELECT isnull(1)") { _ => }
    df.show()
    df.explain(false)
  }

  ignore("Test df.count()") {
    val df = runQueryAndCompare("select * from lineitem limit 1") { _ => }
    df.count()
    df.explain(false)
  }
```

**Comment by FelixYBW** [comment]
Is this part of second phase?

https://github.com/oap-project/gluten/issues/374

**Comment by zhixingheyi-tian** [comment]
The first ignore UT is one common issue, as long as without "from table",  regardless of any expressions/functions.
The second ignore UT is similar to "DataFrameXXXSuite Test Cases:" in #374


cc @zzcclp , Any thoughts?

**Comment by FelixYBW** [comment]
Let's fix this firstly. Just curious if they will be covered by Spark UT. Looks the first one isn't covered.

**Comment by zzcclp** [comment]
Sounds good. We need to covered some common test cases for Gluten, both for Velox backend and CH backend.

**Comment by zhixingheyi-tian** [comment]
Now, first UT was fixed by #564, one workaround way.
Second UT was also passed now.

**Comment by jinchengchenghh** [comment]
Can we close this issue? @zhixingheyi-tian
